.eslintrc.js

```diff
@@ -39,5 +39,6 @@ module.exports = {
   ],
   rules: {
     "react/react-in-jsx-scope": "off",
+    "react/prop-types": "off",
```

Comment:
Being in full strict Typescript, react's prop-type check adds nothing for us

>  For some applications, you can use JavaScript extensions like Flow or TypeScript to typecheck your whole application. 
[React Docs: Typechecking With PropTypes]https://reactjs.org/docs/typechecking-with-proptypes.html